packages/basti/src/aws/elasticache/get-elasticache-memcached-clusters.ts

```diff
@@ -0,0 +1,90 @@
+import { DescribeCacheClustersCommand } from '@aws-sdk/client-elasticache';
+
+import { AwsNotFoundError } from '../common/aws-errors.js';
+
+import {
+  parseMemcachedCacheClusteWithSpecificNoderResponse,
+  parseMemcachedCacheClusterResponse,
+} from './parse-elasticache-memcached-response.js';
+import { elasticacheClient } from './elasticache-client.js';
+
+import type { CacheCluster } from '@aws-sdk/client-elasticache';
+import type { AwsElasticacheMemcachedCluster } from './elasticache-types.js';
+
+export interface getCacheClusterInput {
+  identifier: string;
+}
+
+export async function getMemcachedClusters(): Promise<
+  AwsElasticacheMemcachedCluster[]
+> {
+  const { CacheClusters } = await elasticacheClient.send(
```

Comment:
Why do you make this request if you don't use the response and use the `getRawMemcachedClusters` method instead?